src/ui/styles/apps/file-preview.css

```
@@ -39,20 +38,20 @@
+  transition: color 150ms ease;
 }
 
-.compact-row--ready:hover { color: var(--text); }
-.compact-row--ready:hover .compact-chevron { color: var(--text); }
+.compact-row--ready:hover { color: var(--text-secondary); }
+.compact-row--ready:hover .compact-chevron { color: var(--text-secondary); }
 
 .compact-chevron {
   width: 14px;
   height: 14px;
   fill: currentColor;
```

Stylelint (17.2.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)